traffic_ops/traffic_ops_golang/server/servers.go

```diff
@@ -51,6 +51,12 @@ import (
 const serverCountQuery = `
 SELECT COUNT(s.id)
 FROM server AS s
+JOIN cachegroup cg ON s.cachegroup = cg.id
```

Comment:
Since this is the exact same `FROM ... JOIN`s as the main `selectQuery` below, should we move this into a constant and share them between both queries?

Reply:
I don't really have an opinion, but I suppose it could help avoid something like this in the future. That's a good enough reason to do it.